src/DataManagement/Entries/IEntry.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using ePiggyWeb.DataManagement.Goals;
+
+namespace ePiggyWeb.DataManagement.Entries
+{
+    public interface IEntry : IGoal
```

Comment:
That's a good point, and I'm really not sure on the best practice here. My reasoning was that at the core, these two types are very connected, and since we want to have the feature of converting a goal into expense, this kind of neatly connects the two.
At the same time, since these are just interfaces and not concrete classes, it wouldn't take too much time to implement this change if the need arose.
I'm personally leaning into keeping this as is, but If you think this should be amended I don't mind doing so